### ISSUE TITLE
Red

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/ecosystem-unbitrot#514]
Module [Red](https://modules.perl6.org/dist/Red) cannot be installed (`Fail`), perhaps it has some failing tests.

* Tickets are opened/closed in this repo automatically (though not immediately).
* If you can **install the module without any problems**, add `works for me` label, leave a comment saying that it works for you and mention any details that you feel are important.
* If it **needs a native library**, put `native dependency` label, describe what you did to install it and ensure that same instructions are present in the README file of the module (otherwise submit a pull request). Also try to update [this wiki page](https://github.com/perl6/ecosystem-unbitrot/wiki).
* If the **module is broken**, try to fix it and send a PR. Add `PR sent` label.
* If there is a **problem in one of the dependencies**, add `failing dependency` label and write a comment explaining the situation. Feel free to work on the corresponding ticket for the failing dependency.
* It is a good idea to **assign yourself** to this ticket if you're working on it (to make sure two or more people are not working on the same ticket at the same time).
* **Once you are done**, [search for a next ticket](https://github.com/perl6/ecosystem-unbitrot/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22PR+sent%22+-label%3A%22deprecated+module%22+-label%3A%22native+dependency%22).

If you can't self-assign yourself or attach a label, please let us know on [#perl6 channel on freenode](https://perl6.org/irc) or just leave a comment here. We will try to give you privileges as fast as possible.

Output:

````````````````````````````````````````````````````````
===> Searching for: Red
===> Found: Red:ver<0.0.2>:api<0> [via Zef::Repository::Ecosystems<cpan>]
===> Fetching [OK]: Red:ver<0.0.2>:api<0> to /home/alex/Blin/data/zef-data/tmp/1549583791.29731.1898/Red-0.0.2.tar.gz
===> Extraction [OK]: Red to /home/alex/Blin/data/zef-data/store/Red-0.0.2.tar.gz
===> Testing: Red:ver<0.0.2>:api<0>
t/00-meta.t ........................ ok
t/01-basic.t ....................... ok
t/02-tdd.t ......................... ok
t/03-sqlite.t ...................... ok
t/04-blog.t ........................ ok
t/05-ticket.t ...................... ok
MoarVM panic: Trying to unwind over wrong handler
t/06-better-map.t .................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
t/07-optimizer.t ................... ok
t/08-best-tree.t ................... ok
t/09-alternate-relation.t .......... ok
t/10-alternate-relation-modules.t .. ok
t/11-join.t ........................ ok
t/12-types.t ....................... ok
t/14-result-seq-update.t ........... ok

Test Summary Report
-------------------
t/06-better-map.t                (Wstat: 256 Tests: 0 Failed: 0)
  Non-zero exit status: 1
  Parse errors: Bad plan.  You planned 1 tests but ran 0.
Files=14, Tests=251, 146 wallclock secs ( 0.12 usr  0.01 sys + 163.55 cusr  7.33 csys = 171.01 CPU)
Result: FAIL
===> Testing [FAIL]: Red:ver<0.0.2>:api<0>
Failed to get passing tests, but continuing with --force-test
===> Installing: Red:ver<0.0.2>:api<0>
===> Install [OK] for Red:ver<0.0.2>:api<0>

````````````````````````````````````````````````````````

Ping @FCO

**Comment by FCO** [comment]
I had the same problem when I first tested, than I `rakudobrew build moar`, tried again and everything worked...

**Comment by jonathanstowe** [comment]
The

```
MoarVM panic: Trying to unwind over wrong handler
```

Has got to be a rakudo bug, at the very least an LTA exception,  But yeah I thought this went away a week or so ago.

**Comment by AlexDaniel** [comment]
Nah, it didn't go anywhere. There's still a ticket for it: https://github.com/rakudo/rakudo/issues/2665
